rules/why-your-business-should-be-on-google-my-business/rule.md

```diff
@@ -7,22 +7,25 @@ authors:
   - title: Camilla Rosa Silva
     url: https://www.ssw.com.au/people/camilla-rosa-silva
 created: 2022-05-10T23:09:37.336Z
+related:
+  - optimize-google-my-business-profile
 guid: e92c71d8-38a9-4d30-b363-78841fc9a538
+
 ---
+
 Google My Business is a business listing platform provided by Google. Business owners can create or claim a previously created listing of their business.
 The goal of a Google My Business (GMB) profile is to have your business known by local users (caused by brand awareness), and ultimately, increase your sales.
 
-By adding your business information to Google, you will:
+<!--endintro-->
 
-* Have your business show up on Google Maps;
-* Provide important information to searchers (aka potential customers);
-* Receive business reviews and build trust with your clientele.
+By adding your business information to Google, you will:
 
-<!--endintro-->
+* Have your business show up on Google Maps
+* Provide important information to searchers (aka potential customers)
+* Receive business reviews and build trust with your clientele
 
-If don’t have a Google My Business (GMB) profile yet, it’s unlikely your business will show up on Google Maps and be searchable by Google users. A GMB helps ensure that people find your business when looking for products and services like yours in their local area.
+If don’t have a Google My Business profile yet, it’s unlikely your business will show up on Google Maps and be searchable by Google users. A GMB helps ensure that people find your business when looking for products and services like yours in their local area.
```

Comment:
```suggestion
If don’t have a Google My Business profile yet, it’s unlikely your business will show up on Google Maps and be searchable by Google users. A GMB profile helps ensure that people find your business when looking for products and services like yours in their local area.
```

or:

```suggestion
If don’t have a Google My Business profile yet, it’s unlikely your business will show up on Google Maps and be searchable by Google users. GMB helps ensure that people find your business when looking for products and services like yours in their local area.
```

Note that if you have any poms they may relate GMB to Good Morning Britain (a breakfast show in the UK)